packages/components/src/panel/body.js

```diff
@@ -39,21 +48,26 @@ export function PanelBody(
 		onToggle( next );
 	};
 
+	// Ref is used so that the effect does not re-run upon scrollAfterOpen changing value
+	const scrollAfterOpenRef = useRef();
+	scrollAfterOpenRef.current = scrollAfterOpen;
```

Comment:
I think this could be simplified to:
```suggestion
	const scrollAfterOpenRef = useRef( scrollAfterOpen );
```

And yeah, this definitely feels like a code smell.